waku/v2/protocol/content_topic.go

```diff
@@ -7,7 +7,6 @@ import (
 	"strings"
 )
 
-// DefaultContentTopic is the default content topic used in Waku network if no content topic is specified.
 const DefaultContentTopic = "/waku/2/default-content/proto"
```

Comment:
I don't think this is used or required anymore other than in test code. 
We can move this to test code